irohad/multi_sig_transactions/impl/gossip_propagation_strategy.cpp

```diff
@@ -22,15 +22,15 @@ namespace iroha {
   GossipPropagationStrategy::GossipPropagationStrategy(
       PeerProviderFactory peer_factory,
       rxcpp::observe_on_one_worker emit_worker,
-      std::chrono::milliseconds period,
-      uint32_t amount)
+      const GossipPropagationStrategyParams &params)
       : peer_factory(peer_factory),
         non_visited({}),
         emit_worker(emit_worker),
-        emitent(rxcpp::observable<>::interval(steady_clock::now(), period)
-                    .map([this, amount](int) {
+        emitent(rxcpp::observable<>::interval(steady_clock::now(),
+                                              params.period)
+                    .map([this, params](int) {
                       PropagationData vec;
-                      auto range = boost::irange(0u, amount);
+                      auto range = boost::irange(0u, params.amount);
```

Comment:
amount quite misleading name (at least for non-internal type), maybe you have a better idea for that?

Reply:
I do not think it is misleading. It is the amount of items being output per round.

Reply:
It is. But it isn't implied from the `params.amount`. The one may only deduce "some amount parameter" from that

Reply:
changed the names (see 9611ee1)